cypress/e2e/Mock User to Test and Project.cy.ts

```diff
@@ -0,0 +1,37 @@
+describe('Mocking User and Test Project', () => {
+  it('Mock User and Test Project', () => {
+
+    // Stub login request
+    cy.intercept('GET', 'http://localhost:4173/api/v1/user', {
+      statusCode: 200,
+      body: {
+        id: '756dae7c-6677-4ba3-8fb5-863bcea1b18c',
+        name: 'Cypress Hill',
+        email: 'Cypress@Hill.de',
+        registerDate: '2024-06-11',
+        lasLoginDate: '2024-06-11T15:54:23'
+      }
+    });
+
+    // Other requests
+    // Stub projects feature
+    cy.intercept('GET', 'http://localhost:4173/api/v1/projects', {
+      statusCode: 200,
+      body: {
+        first: 0,
+        size: 1,
+        total: 1,
+        projects: [
+          {
+            id: 'df3d6629-257b-46dc-bbbe-7d3605dd4e03',
+            name: 'TEST',
+            memberRole: 'MANAGER'
+          }
+        ]
+      }
+    });
+
+    // Visit Remsfal page
+    cy.visit('/');
+  });
```

Comment:
Include checks to ensure the UI displays the mocked data correctly like: cy.contains('Cypress Hill').should('be.visible');

Reply:
While stubbing itself is part of testing, it primarily tests the interaction between the front-end code and the API. It ensures that the front-end makes the right requests and processes the responses accordingly. However, stubbing alone does not verify that the UI correctly displays the expected data, as you correctly noted.
So I could include those tests, but I don't know if that's part of the Issue #108, since the Issue itself only concerns creating the stubs.
Furthermore, my original idea for this problem was to use the stubs and use them as custom methods for further test methods.
Maybe @astanik can say what he thinks about this situation.